testing/php7-ssh2/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Andy Postnikov <apostnikov@gmail.com>
+# Maintainer: Andy Postnikov <apostnikov@gmail.com>
+pkgname=php7-ssh2
+_pkgreal=ssh2
+pkgver=1.0
+pkgrel=0
+pkgdesc="PHP extension provide bindings for the libssh2 library"
+url="http://pecl.php.net/package/ssh2"
+arch="all"
+license="PHP"
+depends=""
+pecldepends="php7-dev autoconf"
+makedepends="$pecldepends libssh2-dev"
+source="http://pecl.php.net/get/$_pkgreal-$pkgver.tgz"
```

Comment:
There’s a great chance of a filename conflict, `ssh2-1.0.tgz` is much specific name. I’ve already fixed few package with such problem.

Please change it to `$pkgname-$pkgver`, i.e.:

```diff
- source="http://pecl.php.net/get/$_pkgreal-$pkgver.tgz"
+ source="$pkgname-$pkgver.tgz::http://pecl.php.net/get/$_pkgreal-$pkgver.tgz"
```